Sentry.podspec

```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.frameworks = 'Foundation'
   s.libraries = 'z', 'c++'
+  s.swift_versions = "5.0"
```

Comment:
`h`: Can't we already choose a higher Swift version?

Reply:
Sure, I dont see why not.